Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -1498,9 +1498,11 @@ void WebProcessPool::handleMemoryPressureWarning(Critical)
     m_backForwardCache->clear();
     m_webProcessCache->clear();
 
+    for (WeakPtr maybePrewarmedProcess : copyToVector(m_prewarmedProcesses)) {
```

Comment:
Can we use WeakHashSet::forEach instead?

Reply:
I actually did that originally but then I read the comment in `WeakHashSet::forEach` which seemed to imply that someone wanted to remove the copy. Not sure if that's true or not?